connection.go

```diff
@@ -127,8 +129,18 @@ func (c *conn) open(buf []byte) error {
 	return err
 }
 
+func (c *conn) writeTLS(data []byte) (n int, err error) {
+	// use tls to encrypt the data before sending it.
+	// tlsconn will call gnet.WriteTCP() to sent the data directly.
+	// If gnetConn.outboundBufferis not empty, data will be
```

Comment:
gnetConn.outboundBufferis --> gnet.Conn.outboundBuffer is

Reply:
Resolved in new commits